Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.yml

```diff
@@ -1127,6 +1127,14 @@ script:
       name: roleSessionDuration
       required: false
       secret: false
+    - name: returnRawResponse
+      description: Whether to include all fields from the response in the context, or to store complex fields as JSON for further breakdown as desired. Default value is False.
```

Comment:
```suggestion
      description: Select 'true' to save all fields from the response to the context. Otherwise, complex fields will be stored in JSON format. Default value is False.
```